src/component/TimeLayerSliderPanel/TimeLayerSliderPanel.tsx

```diff
@@ -0,0 +1,353 @@
+import * as React from 'react';
+import {
+  TimeSlider,
+  timeLayerAware,
+  ToggleButton,
+  SimpleButton
+} from '@terrestris/react-geo';
+import {
+  Select,
+  DatePicker,
+  Popover
+} from 'antd';
+
+const { RangePicker } = DatePicker;
+const Option = Select.Option;
+
+const _isFinite = require('lodash/isFinite');
+
+import './TimeLayerSliderPanel.less';
+import moment from 'moment';
+
+type timeRange = [moment.Moment, moment.Moment];
+
+export interface DefaultTimeLayerSliderPanelProps {
+  className: string;
+  onChange: (arg: moment.Moment) => void;
+  timeAwareLayers: any[];
+}
+
+export interface TimeLayerSliderPanelProps extends Partial<DefaultTimeLayerSliderPanelProps> {
+  map: any;
+}
+
+export interface TimeLayerSliderPanelState {
+  value: moment.Moment;
+  forceUpdate: number;
+  playbackSpeed: string;
+  autoPlayActive: boolean;
+  startDate: moment.Moment;
+  endDate: moment.Moment;
+};
+
+/**
+* The panel combining all time slider related parts.
+*/
+export class TimeLayerSliderPanel extends React.Component<TimeLayerSliderPanelProps, TimeLayerSliderPanelState> {
+
+  private _TimeLayerAwareSlider: any;
+  private _wmsTimeLayers: any[];
+  private _interval: number;
+
+  /**
+  * The default props of LayerSetBaseMapChooser
+  *
+  * @static
+  * @type {DefaultLayerSetBaseMapChooserProps}
+  * @memberof LayerSetBaseMapChooser
+  */
+  public static defaultProps: DefaultTimeLayerSliderPanelProps = {
+    className: '',
+    onChange: () => { },
+    timeAwareLayers: []
+  };
+
+  /**
+  * Constructs time panel.
+  */
+  constructor(props: TimeLayerSliderPanelProps) {
+    super(props);
+
+    // this.sunRiseLayer = new SunRiseLayer({
+    //   map: this.props.map
+    // });
+
+    this.state = {
+      value: moment("2015-01-01"),
+      forceUpdate: 1,
+      playbackSpeed: '1',
+      autoPlayActive: false,
+      startDate: moment("2006-01-01"),
```

Comment:
Do we always want to have the same start and current date values regardless of the layer? These values should besser depend on real data I think..